README.md

```diff
@@ -19,11 +21,26 @@ Winfetch currently supports Windows 10; Windows 7 and Windows 8.1 have not been
 
 While Winfetch does work in the majority of cases, things may break occasionally. Please report all such bugs [here](https://github.com/lptstr/winfetch/issues/new).
 
+<br /><br />
+
+## Why use Winfetch?
 <img src="https://lptstr.github.io/lptstr-images/screenshots/projects/winfetch/computant-hyper.png" alt="winfetch" align="right" height="230px" width="350">
 
-**Why use Winfetch?**
 1. **Winfetch is fast.** According to [benchmarks](https://github.com/lptstr/winfetch/wiki/Winfetch-vs-Neofetch) done with Hyperfine, Winfetch is faster than Neofetch running on Git Bash.
 2. **Winfetch isn't bloated.** The whole thing is only 12.5KB and 380 lines of pure PowerShell, counting comments. Compare that to Neofetch, which is about 280KB and 7,500 lines of code.
 3. **No need for the WSL, Cygwin, or MSYS2.** While you need to install a bash emulator (WSL, Git Bash, MSYS2, Cygwin) to run Neofetch, all you need for Winfetch is PowerShell, which is installed by default on most Windows computers.
 
+
+## Quick Usage
+
+```pwsh
+$> Install-Script posh-winfetch -Scope CurrentUser`
+$> posh-winfetch
+```
+
+## Contributors
+
+* [@lptstr](github.com/lptstr)
+* [@ppadial](github.com/ppadial)
+
```

Comment:
I'd prefer if installation/contributor information stays in the wiki.

Reply:
Well, it's fine either way for me, i used it from my Open Source README.md skeleton, most projects have it on the main screen.

In fact, we have to add to the people that has contribute, not just you and me.

Reply:
I've removed it but since i dont have access to it, you need to add teh page in wiki and add previous contributors to it, i will add a link to the wiki page.